pkg/oauth/apis/oauth/v1/types.go

```diff
@@ -37,6 +37,9 @@ type OAuthAccessToken struct {
 
 	// RefreshToken is the value by which this token can be renewed. Can be blank.
 	RefreshToken string `json:"refreshToken,omitempty" protobuf:"bytes,9,opt,name=refreshToken"`
+
+	// TimeoutsIn is the seconds from CreationTime before this token timeouts.
```

Comment:
Not sure if this doc string makes sense.

Reply:
It's the same as for ExpiresIn, copied word by word :)

Reply:
Right, but its true for ExpiresIn, and false for TimeoutsIn.

Reply:
what ?

Reply:
Ignore this set of comments 😄